packages/url_launcher/url_launcher/android/gradle.properties

```diff
@@ -1 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
Should this two lines also be added to example/android/gradle.properties file? Also, as added to that file, should R8 be enabled here too?

(sorry if it's too late)